local.go

```diff
@@ -260,11 +260,11 @@ func (l *LocalTest) NewTreeNodeInstance(tn *TreeNode, protName string) (*TreeNod
 	return node, nil
 }
 
-// getNodes returns all Nodes that belong to a treeNode
-func (l *LocalTest) getNodes(tn *TreeNode) []*TreeNodeInstance {
+// GetTreeNodeInstances returns all TreeNodeInstances that belong to a server
+func (l *LocalTest) GetTreeNodeInstances(id network.ServerIdentityID) []*TreeNodeInstance {
```

Comment:
Why does this need to be public?

Reply:
so that tests can ask what treenodeinstances are still alive.

In a second step I would like to add something like the following to `LocalTest.CloseAll`:

```
	for _, s := range o.local.Servers {
		for _, tn := range o.local.GetTreeNodeInstances(s.ServerIdentity.ID) {
			log.Lvl1("Still active: ", tn.Info())
		}
	}
```